extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -126,6 +114,23 @@ public function testRenderInfoboxDataProvider() {
 			[
 				'input' => [
 					[
+						'type' => 'title',
+						'data' => [
+							'value' => 'Test Title'
+						]
+					]
+				],
+				'output' => '<aside class="portable-infobox pi-background">
+								<h2 class="pi-item pi-item-spacing pi-title" style="background-color:#FFF;color:#000;">Test Title</h2>
+							</aside>',
+				'description' => 'Only title with custom colors',
+				'mockParams' => [],
+				'accentColor' => '#FFF',
+				'accentColorText' => '#000'
+			],
+			[
+				'input' => [
+					[
 						'type' => 'image',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "image" 16 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)